modules/dynamic/test/src-jvm/smithy4s/dynamic/package.scala

```diff
@@ -0,0 +1,28 @@
+package smithy4s
+
+import smithy4s.dynamic.DummyIO._
+import java.nio.file.Paths
+import software.amazon.smithy.model.{Model => SModel}
+import software.amazon.smithy.model.shapes.ModelSerializer
+import cats.syntax.all._
+
+package object dynamic {
+  // This is not ideal, but it does the job.
+  private val cwd = System.getProperty("user.dir");
```

Comment:
make this lazy : running effects during instantiation of static objects (including package objects) is a big no-no

Reply:
Good catch, will do!